esp8266-fastled-webserver/config.h

```diff
@@ -0,0 +1,181 @@
+/*
+   ESP8266 FastLED WebServer: https://github.com/jasoncoon/esp8266-fastled-webserver
+   Copyright (C) Jason Coon
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#if !defined(ESP8266_FASTLED_WEBSERVER_CONFIG_H)
+#define ESP8266_FASTLED_WEBSERVER_CONFIG_H
+
+
+// This header file lists (and provides defaults for) the various configurations.
+// When compiling from Arduino, you can edit this file.
+// When compiling from PlatformIO, this is a reference for compiler flags.
+
+#define PRODUCT_DEFAULT
```

Comment:
SIMPLE FILE REVIEW

This is the file where the product to be built is defined.
Simply uncommenting the product symbols define here
will change which product's binary is being built.


The remainder of the file is just sanity-checking of the various
symbols defined by the product and controller header files
(see `include/configs/product/template.h` for details).